src/main/java/domain/result/DealerResult.java

```diff
@@ -1,23 +1,25 @@
 package domain.result;
 
-import domain.WinState;
-import java.util.Collections;
-import java.util.LinkedHashMap;
+import domain.Profit;
+import domain.gamer.Player;
 import java.util.Map;
 
 public class DealerResult {
 
-    private final Map<WinState, Integer> result;
+    private final Profit profit;
 
     public DealerResult() {
-        this.result = new LinkedHashMap<>();
+        this.profit = new Profit(0);
     }
 
-    public void addResult(WinState winState, int winStateCount) {
-        result.put(winState, winStateCount);
+    public void calculateProfit(PlayersResult playersResult) {
+        Map<Player, Profit> playersProfit = playersResult.getPlayersProfit();
+        profit.update(-playersProfit.values().stream()
+                .mapToDouble(Profit::getValue)
+                .sum());
```

Comment:
가독성이 떨어져요. `-` 연산자와 합 로직을 분리해주세요.
```suggestion
         var profitSum = playersProfit.values().stream()
                .mapToDouble(Profit::getValue)
                .sum()
        profit.update(-profitSum);
```